src/dune_rules/ordered_set_lang.mli

```diff
@@ -55,6 +57,10 @@ module Unexpanded : sig
 
   val of_strings : pos:string * int * int * int -> string list -> t
 
+  val standard_with_of_strings : pos:string * int * int * int -> string list -> t
```

Comment:
I'd change this to a `Loc.t` argument. If you want to pass the current position in the file, then you have the macro `Loc.of_pos __POS__`.